models/rubicon/arbitrum/rubicon_arbitrum_trades.sql

```diff
@@ -1,6 +1,7 @@
 {{ config(
     schema = 'rubicon_arbitrum',
     alias = alias('trades'),
+    tags = ['dunesql'],
```

Comment:
Hi @denverbaumgartner, I think we need to migrate this spell first.
I just added a dunesql tag,
but I am not sure if this spell has been migrated yet.